Looks like golint is complaining about the period at the end;

```
01:22:11 ---> Making bundle: validate-lint (in bundles/1.13.0-dev/validate-lint)
01:22:11 Errors from golint:
01:22:11 distribution/pull_v2.go:836:21: error strings should not end with punctuation
01:22:11 
01:22:11 Please fix the above errors. You can test via "golint" and commit the result.
```

ping @YuPengZTE can you fix that?

--- a/distribution/pull_v2.go
+++ b/distribution/pull_v2.go
@@ -810,7 +810,7 @@ func fixManifestLayers(m *schema1.Manifest) error {
 
 	if imgs[len(imgs)-1].Parent != "" && runtime.GOOS != "windows" {
 		// Windows base layer can point to a base layer parent that is not in manifest.
-		return errors.New("Invalid parent ID in the base layer of the image.")
+		return errors.New("invalid parent ID in the base layer of the image")
 	}
 
 	// check general duplicates to error instead of a deadlock

--- a/distribution/pull_v2_test.go
+++ b/distribution/pull_v2_test.go
@@ -80,7 +80,7 @@ func TestFixManifestLayersBaseLayerParent(t *testing.T) {
 		},
 	}
 
-	if err := fixManifestLayers(&duplicateLayerManifest); err == nil || !strings.Contains(err.Error(), "Invalid parent ID in the base layer of the image.") {
+	if err := fixManifestLayers(&duplicateLayerManifest); err == nil || !strings.Contains(err.Error(), "invalid parent ID in the base layer of the image") {
 		t.Fatalf("expected an invalid parent ID error from fixManifestLayers")
 	}
 }

